### ISSUE TITLE
Secret.get does not respect option additionalSecretOutputs

### ISSUE THREAD

**Issue by amkartashov** [pulumi/pulumi-kubernetes#956]
### Problem description

I read existing k8s secret using static method https://www.pulumi.com/docs/reference/pkg/nodejs/pulumi/kubernetes/core/v1/#Secret-get

To make output secure, I add option `additionalSecretOutputs` as per https://www.pulumi.com/docs/intro/concepts/programming-model/#additionalsecretoutputs but output is displayed in plain text every invocation of `pulumi up` or `pulumi stack output` nevertheless.

Code:

```
const vaultAuthAccount = new k8s.core.v1.ServiceAccount(
...
// read from created account token secret
export const vaultAuthToken64 = k8s.core.v1.Secret.get(
  "vault-auth-token",
  pulumi.interpolate`${vaultAuthAccount.metadata.namespace}/${vaultAuthAccount.secrets[0].name}`,
  {
      additionalSecretOutputs: ["data"]
  }
).data.apply(d => (d as { token: string }).token);
```

Actually, I expect `data` and `stringData` to be secure outputs by default, without specifying additionalSecretOutputs.

### Affected product version(s)

pulumi version 1.8.1
pulumi-kubernetes version 1.4.3

### Workaround

Wrap output into `pulumi.secret()` to make it secure

**Comment by pgavlin** [comment]
@lblackstone can you help me get a minimal repro for this?

**Comment by lblackstone** [comment]
@pgavlin Here's a minimal repro:

```ts
const secret = new k8s.core.v1.Secret("secret", {
    metadata: {name: "secret"},
    stringData: {password: "super-secret-password-value"}
});

// Uncomment after running `pulumi up`; base64-encoded secret value will be visible
// on next preview/update

//export const secretVal = k8s.core.v1.Secret.get('test-secret', 'default/secret', {
//    additionalSecretOutputs: ["data"]}).data.apply((d: {password: string}) => d.password);
```

**Comment by lblackstone** [comment]
![Screen Shot 2020-01-22 at 10 52 18 AM](https://user-images.githubusercontent.com/9253463/72919758-49c50180-3d05-11ea-9584-34bfdb6ec9c9.png)

Checked the generated state, and it appears that `data` is properly being marked as secret, but is still appearing in plaintext on the annotation if the input value was not marked as secret.

**Comment by lblackstone** [comment]
This was probably fixed quite awhile ago, but #2444 definitely fixes if not.
